tests/admin_views/tests.py

```diff
@@ -6,6 +6,8 @@
 from unittest import mock
 from urllib.parse import parse_qsl, urljoin, urlparse
 
+from selenium.webdriver import ActionChains
```

Comment:
`selenium` is an optional requirement, please move this import to the test method.

Reply:
Great catch, thank you! fixing and rebasing and squashing and merging :-)